File: planit v0.2/Source/HotelsSource/Utils/MailSender/HLMailComposeVC.m
//
//  HLMailComposeVC.m
//  HotelLook
//
//  Created by Anton Chebotov on 23/04/14.
//  Copyright (c) 2014 Anton Chebotov. All rights reserved.
//

#import "planit_v0_2-Swift.h"
#import "HLMailComposeVC.h"

@implementation HLMailComposeVC

- (void) viewWillAppear:(BOOL)animated
{
    [self.parentViewController resignFirstResponder];
    [super viewWillAppear:animated];
	
	[self.view setNeedsLayout];
	[self.view layoutIfNeeded];
}

@end

File: planit v0.2/HotelsSource/Utils/MailSender/HLMailComposeVC.swift
//  Converted with Swiftify v1.0.6402 - https://objectivec2swift.com/
//
//  HLMailComposeVC.swift
//  HotelLook
//
//  Created by Anton Chebotov on 23/04/14.
//  Copyright (c) 2014 Anton Chebotov. All rights reserved.
//

import MessageUI

class HLMailComposeVC: MFMailComposeViewController {
    override func viewWillAppear(_ animated: Bool) {
        parentViewController?.resignFirstResponder()
        super.viewWillAppear(animated)
        view.setNeedsLayout()
        view.layoutIfNeeded()
    }
}